### PR TITLE
Feature/map 166/and/map 167/create/user profile/component_and_endpoint

### PR DESCRIPTION
**Pull request by romawht22**
You can click on user name or avatar in both location popup and comment section to visit their profile

### CODE REVIEW

File: frontend/src/components/PersonProfile/PersonProfile.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { useTypedSelector } from 'redux/hooks/useTypedSelector';
+import { useTranslation } from 'react-i18next';
+import PersonProfilePage from './PersonProfilePage';
+
+function PersonProfile() {
+  const { t } = useTranslation();
+  const { error: userError, loading: userLoading } = useTypedSelector(
+    state => state.userData
```
Comment (obozhko-dev) [inline frontend/src/components/PersonProfile/PersonProfile.tsx]: Maybe here should be `state.otherUserData` or its correct ? I don`t understand why we using personal user data here

**Review by obozhko-dev — CHANGES_REQUESTED** [review]
When I open Location that I created i saw that
image

Date crashed in all locations , this is old problem as i remember or not? 
For own locations should be link to PersonProfile or avatar and name should be not clickable

**Comment by romawht22** [comment]
> When I open Location that I created i saw that image
> 
> Date crashed in all locations , this is old problem as i remember or not? For own locations should be link to PersonProfile or avatar and name should be not clickable

I think it is corrupted userData in mongo

File: frontend/src/components/PersonProfile/PersonProfile.tsx (hunk 1 above)
Comment (romawht22) [reply frontend/src/components/PersonProfile/PersonProfile.tsx]: Fixed

**Review by obozhko-dev — APPROVED** [review]

**Review by vasylneshchadimov — APPROVED** [review]

**Pull request merged**